src/fmu/dataio/_model/global_configuration.py

```diff
@@ -108,6 +108,12 @@ class Stratigraphy(RootModel[Dict[str, StratigraphyElement]]):
     A collection of StratigraphyElement instances, accessible by keys.
     """
 
+    def __iter__(self) -> Dict[str, StratigraphyElement]:
+        return iter(self.root)
```

Comment:
Should be the same thing here with returning an `Iterator`. However, we may need to see if mypy likes this or not 😄